include/cantera/thermo/PengRobinson.h

```diff
@@ -0,0 +1,320 @@
+//! @file PengRobinson.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_PENGROBINSON_H
+#define CT_PENGROBINSON_H
+
+#include "MixtureFugacityTP.h"
+#include "cantera/base/Array.h"
+
+namespace Cantera
+{
+/**
+ * Implementation of a multi-species Peng-Robinson equation of state
+ *
+ * @ingroup thermoprops
+ */
+class PengRobinson : public MixtureFugacityTP
+{
+public:
+
+    //! Construct and initialize a PengRobinson object directly from an
+    //! input file
+    /*!
+     * @param infile    Name of the input file containing the phase YAML data.
+     *                  If blank, an empty phase will be created.
+     * @param id        ID of the phase in the input file. If empty, the
+     *                  first phase definition in the input file will be used.
+     */
+    explicit PengRobinson(const std::string& infile="",
+                              const std::string& id="");
+
+    virtual std::string type() const {
+        return "PengRobinson";
+    }
+
+    //! @name Molar Thermodynamic properties
+    //! @{
+
+    virtual double cp_mole() const;
+    virtual double cv_mole() const;
+
+    //! @}
+    //! @name Mechanical Properties
+    //! @{
+
+    //! Return the thermodynamic pressure (Pa).
+    /*!
+     * Since the mass density, temperature, and mass fractions are stored,
+     * this method uses these values to implement the
+     * mechanical equation of state \f$ P(T, \rho, Y_1, \dots, Y_K) \f$.
+     *
+     * \f[
+     *    P = \frac{RT}{v-b_{mix}} - \frac{\left(\alpha a\right)_{mix}}{v^2 + 2b_{mix}v - b_{mix}^2}
+     * \f]
+     *
+     * where:
+     *
+     * \f[
+     *    \alpha = \left[ 1 + \kappa \left(1-T_r^{0.5}\right)\right]^2
+     * \f]
+     *
+     *  and
+     *
+     * \f[
+     *    \kappa = \left(0.37464 + 1.54226\omega - 0.26992\omega^2\right), \qquad \qquad \text{For } \omega <= 0.491 \\
+     *    \kappa = \left(0.379642 + 1.487503\omega - 0.164423\omega^2 +  0.016667\omega^3 \right), \qquad \text{For } \omega > 0.491
+     * \f]
+     *
+     * Coefficients \f$ a_mix, b_mix \f$ and \f$(a \alpha)_{mix}\f$ are calculated as
+     *
+     * \f[
+     *    a_{mix} = \sum_i \sum_j X_i X_j a_{i, j} = \sum_i \sum_j X_i X_j \sqrt{a_i a_j}
+     * \f]
+     *
+     * \f[
+     *    b_{mix} = \sum_i X_i b_i
+     * \f]
+     *
+     * \f[
+     *   {a \alpha}_{mix} = \sum_i \sum_j X_i X_j {a \alpha}_{i, j} = \sum_i \sum_j X_i X_j \sqrt{a_i a_j} \sqrt{\alpha_i \alpha_j}
+     * \f]
+     *
+     *
+     */
+    
+    virtual double pressure() const;
```

Comment:
Generally:
- No blank lines between the docstring and the declaration (this one contains whitespace to top things off)
- No "blank" lines (just containing `*`) at the start or end of the docstring.
```suggestion
     * \f]
     */
    virtual double pressure() const;
```